numba_dppy/tests/integration/__init__.py

```diff
@@ -16,4 +16,12 @@
 # limitations under the License.
 ################################################################################
 
+import pytest
+
+from numba_dppy.tests._helper import ensure_dpnp
+
 from . import *
+
+pytestmark = pytest.mark.xfail(not ensure_dpnp(), reason="No dpnp available")
+if ensure_dpnp():
+    import dpnp
```

Comment:
Use this approach:
https://github.com/IntelPython/numba-dppy/blob/19eb6fe1f402a0de85f7e26986a71c313a1a67cd/numba_dppy/tests/test_debug_dppy_numba.py#L27

Reply:
If we use this, there is no information about which tests and number was skipped:
collected 0 items / 1 skipped